bot_modules/help_command/index.js

```diff
@@ -0,0 +1,30 @@
+exports.init = (client) => {
+  client.on('message', async (msg) => {
+    if (msg.content.startsWith('!help')) {
+      // TODO: Add a command registration system instead of hardcoding
+      msg.channel.send({
+        embed: {
+          title: 'Geyser Bot Help',
+          color: 0x00ff00,
+          fields: [
+            {
+              name: '`!help`',
+              value: 'I think you already know what this does',
+              inline: true
+            },
+            {
+              name: '`!wiki <search>`',
+              value: 'Search the Geyser wiki',
+              inline: true
+            },
+            {
+              name: '`!provider <provider>`',
+              value: 'Search the Supported Providers page on the Geyser wiki to see if a provider is supported',
+              inline: true
+            }
+          ]
+        }
+      })
```

Comment:
This should be converted to use the `Discord.MessageEmbed` class, to allow for easier dynamic generation later
```javascript
const helpEmbed = new Discord.MessageEmbed()
  .setColor('#00FF00')
  .setTitle('Geyser Bot Help')

helpEmbed.addField('`!help`', 'I think you already know what this does', true)
helpEmbed.addField('`!wiki <search>`', 'Search the Geyser wiki', true)
helpEmbed.addField('`!provider <provider>`', 'Search the Supported Providers page on the Geyser wiki to see if a provider is supported', true)

msg.channel.send(helpEmbed)
```

Reply:
Done